BaseFilter/sections/specific.txt

```diff
@@ -907,7 +907,7 @@ boards.4channel.org,boards.4chan.org##.adc-resp-bg
 ||fmmods.com/bnbanner/
 fmmods.com##.rbc-container > center > h5
 xero.porn###popad
-xero.porn##div[onclick^="$\(\"#videoAd"]
+xero.porn##div[onclick^="$(\"#videoAd"]
```

Comment:
This change is strange, but looks like that this rule is obsolete. Could you remove it?

Reply:
Removed

Reply:
the rule is not obsolete, and modified rule is still working when `"` is only escaped.

Reply:
Also no need to check for obsolete rules in this PR.

Reply:
I got it, but where the rule is needed?
Anyway, looks ok.

Reply:
On the page with player.